src/app/Home/components/HomeComponent.js

```diff
@@ -1,14 +1,71 @@
 /* eslint-disable react/destructuring-assignment */
 import React, { Component } from 'react';
-import TestSaga from './testSaga';
+import Modal from './onBoardingModal';
+import LoginModal from './logInPointsModal';
 
 // eslint-disable-next-line react/prefer-stateless-function
 export class HomeComponent extends Component {
-  render() {
-    return (
-      <TestSaga />
-    );
+  constructor() {
+    super();
+
+    this.state = {
+      isShowing: false,
+      isShowingLoginPoints: false,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      isShowing: true,
+    });
   }
+
+closeModalHandler = () => {
+  this.setState({
+    isShowing: false,
+  });
+}
+
+openModalLoginPointsHandler = () => {
+  this.setState({
+    isShowingLoginPoints: true,
+  });
+}
+
+closeModalLoginPointsHandler = () => {
+  this.setState({
+    isShowingLoginPoints: false,
+  });
+}
```

Comment:
We can use one function you don't need closeModalLoginPointsHandler, openModalLoginPointsHandler

Reply:
actually, I don't think you even need closeModalHandler